app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,71 @@
+require 'date'
```

Comment:
We don't need to require the `date` library here, because Rails already provides us with the `Date` class (and many other related date and time classes).

In general we shouldn't need to explicitly `require` any libraries with Rails because it has some very fancy logic that looks at our `Gemfile` and figures out what to require for us.